crates/starknet-devnet/tests/test_get_events.rs

```diff
@@ -43,6 +42,10 @@ mod get_events_integration_tests {
 
         let predeployed_account = Arc::new(predeployed_account);
 
+        if block_on_demand {
+            let _ = devnet.create_block().await;
```

Comment:
Here and in other places, I'd avoid `let _ =`, but I'd unwrap:
```rust
devnet.create_block().await.unwrap();
```

Reply:
bf091b7494899019ace95a45775796b445133716